src/Thentos/Smtp.hs

```diff
@@ -6,6 +6,7 @@ module Thentos.Smtp
     ( sendUserConfirmationMail
     , sendUserExistsMail
     , sendPasswordResetMail
+    , sendEmailChangeConfirmMail
```

Comment:
How so? It's `send Password-reset mail`, why not `send email-change confirm mail`? (Although it should be "confirmation", not "confirm")